packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -130,10 +132,20 @@ export class CdkToolkit {
       if (!await fs.pathExists(options.templatePath)) {
         throw new Error(`There is no file at ${options.templatePath}`);
       }
+
+      const changeSet = options.changeSet ? await PrepareAndCreateDiffChangeSet({
```

Comment:
```suggestion
      const changeSet = options.changeSet ? await prepareAndCreateDiffChangeSet({
```

What is the difference between the verbs `prepare` and `create` ?

Reply:
`prepare` means we're preparing an SDK with the correct role, making the body parameter, and checking CFN to see if the stack exists. 

`create` is just calling `CreateChangeSet` and cleaning up the changeset

Reply:
Prepare is not useful to the caller of this function though, is it?